Capstone Project/KASlideShow/KASlideShow.h

```diff
@@ -0,0 +1,76 @@
+//
+//  KASlideShow.h
+//
+
+#import <UIKit/UIKit.h>
+
+typedef NS_ENUM(NSInteger, KASlideShowTransitionType) {
+    KASlideShowTransitionFade,
+    KASlideShowTransitionSlideVertical,
+    KASlideShowTransitionSlideHorizontal
+};
+
+typedef NS_ENUM(NSInteger, KASlideShowGestureType) {
+    KASlideShowGestureTap,
+    KASlideShowGestureSwipe,
+    KASlideShowGestureAll
+};
+
+typedef NS_ENUM(NSUInteger, KASlideShowPosition) {
+    KASlideShowPositionTop,
+    KASlideShowPositionBottom
+};
+
+typedef NS_ENUM(NSUInteger, KASlideShowState) {
+    KASlideShowStateStopped,
+    KASlideShowStateStarted
+};
+
+typedef NS_ENUM(NSInteger, KASlideShowDirection) {
+    KASlideShowDirectionForward,
+    KASlideShowDirectionBackward
+};
+
+@class KASlideShow;
+@protocol KASlideShowDelegate <NSObject>
+@optional
+- (void) slideShowDidShowNext:(KASlideShow *) slideShow;
```

Comment:
Is this framework used for slideshow.  I thought you were planning for a movie like viewing experience